README.md

```diff
@@ -62,6 +64,7 @@ jobs:
           find: "hello"
           replace: "world"
           include: "justthisdirectory\/"
+          regex: false
```

Comment:
Let's not set regex to false everywhere. Just one example is probably fine. `hello` is a perfectly valid regex.